Reflections/ReflectionWeek3.md

```diff
@@ -0,0 +1,15 @@
+1. For this week I have accomplished most of everything that I wanted to do:
+- I fixed my logo, added a blacked gradient background, and changed the colors of my buttons so I could have a consistent theme.
+- I also implemented the spinning wheel, so know when the user finishes the filters section they have a colorful wheel that will be displayed before the restaurant is chosen.
+- I know I can't use it just yet but I also created a System Usability Survey for my app to get feedback from friends and family members that I have told about my project.
+
+2. The biggest challenge that I faced this week was the implementation of the spinning wheel. The links to the youtube tutorials that I have previously listed, were older versions of the thunkable website and couldn't be used. I basically went to the Thunkable forums to try and figure out what was different.(https://community.thunkable.com/t/image-rotation-not-available-in-thunkable-x/1061768/4) Turns out that instead of allowing images to rotate, they made them into sprites on a canvas. This made it easier for anyone using Thunkable to create a mobile game.
+
+3. Now that I have a big majority of my app finished I am not sure what to do now. I did talk with one of my friends over the weekend and we discussed filterting the restaurants by breakfast, lunch, and dinner. However in the Yelp API documentation those don't show up in the categories.

```

Comment:
I am pretty sure you can ask for suggestions in your next report out this  week! Other students, the professors and TAs might have a lot of new ideas to try to implement in your app. 
Also, you can check out similar apps to see what other kind of fun things they did with the Yelp API.